integrations/oci/metrics/cdi/src/main/java/io/helidon/integrations/oci/metrics/cdi/OciMetricsConfigProducer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.oci.metrics.cdi;
+
+import javax.annotation.Priority;
+import javax.enterprise.inject.Produces;
+
+import io.helidon.config.Config;
+
+@Priority(0)
+class OciMetricsConfigProducer {
+
+    @Produces
+    @OciMetrics
+    public Config config(Config config) {
+        return config;
```

Comment:
Double-checking that you want to return `@Default Config` as (also) `@OciMetrics Config`. I guess this is the "backstop" implementation, so that users supplying their own `@OciMetrics Config` will have their stuff picked up first?

Reply:
Yes. I will add a comment explaining that this is intended.